.travis.yml

```diff
@@ -108,22 +102,28 @@ jobs:
                           - docker run --privileged -i -t
                               -v `pwd`:/odp --shm-size 8g
                               -e CODECOV_TOKEN="${CODECOV_TOKEN}"
-                              -e CC="${CC}" -e LD="${LD}" -e CXX="${CXX}"
+                              -e CC="${CC}"
                               -e CONF="${CONF}"
-                              ${DOCKER_NAMESPACE}/travis-odp-lng-x86 /odp/scripts/ci/coverage.sh
+                              ${DOCKER_NAMESPACE}/travis-odp-lng-ubuntu_16.04 /odp/scripts/ci/coverage.sh
```

Comment:
just make attention here if we need lng in naming. Looks like better to switch to travis-odp-ubuntu_16.04.

Reply:
Also, we can change that later.